app/utils/DateTimeFormats.scala

```diff
@@ -28,4 +28,9 @@ object DateTimeFormats {
   val ISO8601Formatter: DateTimeFormatter = DateTimeFormatter
     .ofPattern("yyyy-MM-dd'T'HH:mm:ss'Z'")
     .withZone(ZoneId.of("UTC"))
+
+  val EmailDateTimeFormatter: DateTimeFormatter = DateTimeFormatter
+    .ofPattern("h:mma (z) 'on' d MMMM yyyy")
```

Comment:
Should the timezone have brackets?  And do we want this to always be GMT, as opposed to showing GMT or BST depending on the actual timezone (which is how dates are shown in other parts of the service)?

Reply:
Other parts of the service do display timezone within brackets using GMT:
https://github.com/hmrc/digital-platform-reporting-register-frontend/blob/main/app/models/pageviews/RegistrationConfirmationViewModel.scala#L34

Reply:
I've raised this with Kerry in Slack, just waiting a reply:
https://hmrcdigital.slack.com/archives/C05QVP65A0M/p1732884067793949